### PR TITLE
Add Graphul

### PR DESCRIPTION
**Pull request by SamuelBonilla**
An Express-inspired web framework written in Rust.
https://github.com/graphul-rs/graphul

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
Please add a description or a like to an issue

**Comment by Israel-Laguan** [comment]
Please add a description or a like to an issue.

File: README.md [hunk 1]
```diff
@@ -1525,6 +1525,7 @@ See also [Are we web yet?](https://www.arewewebyet.org) and [Rust web framework
   * [hyperium/hyper](https://github.com/hyperium/hyper) — an HTTP implementation [![CI](https://github.com/hyperium/hyper/workflows/CI/badge.svg?branch=master)](https://github.com/hyperium/hyper/actions?query=workflow%3ACI)
   * [seanmonstar/reqwest](https://github.com/seanmonstar/reqwest) — an ergonomic HTTP Client for Rust. [![build badge](https://api.travis-ci.org/seanmonstar/reqwest.svg?branch=master)](https://travis-ci.org/seanmonstar/reqwest)
 * HTTP Server
+  * [Graphul](https://github.com/graphul-rs/graphul) — An Express-inspired web framework written in Rust. [![crate](https://img.shields.io/crates/v/create-rust-app.svg)](https://crates.io/crates/graphul)
```
Comment (Israel-Laguan) [inline README.md:1528]: Seems like this list should be ordered alphabetically, so please move this entry below.

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]

**Comment by palfrey** [comment]
> Please add a description or a link to an issue this PR solves

Hi. So, in common with most of the PRs here, the issue they solve is "this project isn't linked". Was this comment added automatically?

File: README.md (hunk 1 above)
Comment (SamuelBonilla) [reply README.md:1528]: Ok

**Review by Israel-Laguan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SamuelBonilla) [reply README.md]: Fixed

**Pull request merged**